sysml.library/Domain Libraries/Quantities and Units/UnitsAndScales.sysml

```diff
@@ -1,37 +1,52 @@
 /**
  * This package defines the representations for measurement references.
- * TODO: Propose to rename to MeasurementReferences
+ * TODO: Rename package to MeasurementReferences as this name captures the content much better

```

Comment:
Please do not put "TODOs" in library packages. Any issues or proposals should be recording in Jira for potential scheduling in a sprint.